prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -153,10 +155,18 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
           this.enrolmentFormStateService.setForm(enrolment);
 
           this.enrolmentErrors = this.getEnrolmentErrors(enrolment);
+          // Determine if Enrolment is within 90 days of expiration
+          this.enrolmentWithin90DaysOfExpiry();
         })
       ).subscribe();
   }
 
+  public enrolmentWithin90DaysOfExpiry(): void {
```

Comment:
it seems `enrolmentWithin90DaysOfExpiry` is not called in the template so maybe only need `private` access modifier. other than this, LGTM